.github/workflows/setup-flatcar-sdk.sh

```diff
@@ -2,6 +2,8 @@
 
 set -euo pipefail
 
+sudo apt-get install -y lbzip2
```

Comment:
This may fail depending on whether the OS image is rebuilt often or not and if lbzip2 is updated. A `sudo apt update` first would make it a bit more robust.

Reply:
I'll add a sudo apt update if we ever observe it failing.